internal/controller/scheduler/scheduler.go

```diff
@@ -28,6 +28,12 @@ var (
 	workersStat = promauto.NewGaugeVec(prometheus.GaugeOpts{
 		Name: "orchard_workers",
 	}, []string{"status"})
+	workersStatus = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "orchard_worker_status",
+	}, []string{"worker_name", "status"})
+	workersLastSeen = promauto.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
Could you elaborate a bit on which problem does this metric solve?

Useful for history reasons, and perhaps there's another way to solve the underlying issue.

Reply:
It's not necessarily a problem that I'm trying to resolve, more just it's a metric that would be good to export. 
I suspect that my new metrics above it (orchard_worker_status) will give me everything I need, as I'll be able to alert if it meets the offline criteria. But thought whilst I was here I could offer another metric to users.

Reply:
Appreciate your explanation.

It's best to avoid adding features without a specific need, though. Doing so can make future changes difficult, especially if people start using those features. Without a real-world application to guide the initial design, you might later realize the public interface isn't well-suited for actual use cases.

But since Orchard doesn't have much metrics to worry much at the moment, it's your call whether to keep this or not 🙌

Reply:
Removed it